index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="icon" type="image/x-icon" href="https://s.ytimg.com/yts/img/favicon-vfl8qSV2F.ico" />
+    <title>Youtube Clone</title>
+    <link rel="stylesheet" href="youtubestyle.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <div class="navbar">
```

Comment:
[OPTIONAL]
Using the `nav` tag for the navbar make your HTML code more semantic. This is a very important topic. You can learn more about [here](http://seekbrevity.com/semantic-markup-important-web-design/).